editor/questionbankcontroller.js

```diff
@@ -501,7 +502,7 @@ angular.module('createquestionapp', [])
       delete $scope.selectedQueIndex;
       $scope.selectedQueIndex = index;
       var filterMetaData = {};
-      ecEditor.dispatchEvent('org.ekstep.editcontentmeta:showpopup', { action: 'question-filter-view', subType: 'questions', framework: ecEditor.getContext('framework'), rootOrgId: ecEditor.getContext('channel'), type: 'content', popup: false, metadata: filterMetaData});
+      ecEditor.dispatchEvent('org.ekstep.editcontentmeta:showpopup', { action: 'question-filter-view', subType: 'questions', framework: "NCF", rootOrgId: ecEditor.getContext('channel'), type: 'content', popup: false, metadata: filterMetaData});
```

Comment:
Why have you committed hard-coded values? Please verify your changes before you commit!